meta-multimedia/recipes-multimedia/libcamera/libcamera.bb

```diff
@@ -0,0 +1,20 @@
+SUMMARY = "Linux libcamera framework"
+SECTION = "libs"
+
+LICENSE = "GPL-2.0 & LGPL-2.1"
+LIC_FILES_CHKSUM = "\
+    file://licenses/gnu-gpl-2.0.txt;md5=b234ee4d69f5fce4486a80fdaf4a4263 \
+    file://licenses/gnu-lgpl-2.1.txt;md5=4b54a1fd55a448865a0b32d41598759d \
+"
+
+SRC_URI = "git://linuxtv.org/libcamera.git;protocol=git"
+
+S = "${WORKDIR}/git"
+SRCREV = "${AUTOREV}"
+
+DEPENDS = "python udev qtbase qtbase-native"
+
+inherit meson pkgconfig
+
+FILES_${PN}-dev = "${includedir} ${libdir}/pkgconfig"
+FILES_${PN} += " ${libdir}/*.so"
```

Comment:
do we need .so in main package ? I thought this should be a development file symlink pointing to real shared object

Reply:
libcamera is generating the libcamera.so, and this .so file is needed during the run time.
Now, we had added libcamera.so files instead of *.so